test/common/crudtester.go

```diff
@@ -207,11 +207,20 @@ func (c *FederatedTypeCrudTester) CheckPlacementChange(template, placement, over
 
 	clusterNames := adapter.ClusterNames(placement)
 
+	if kind == federatedtypes.FederatedNamespacePlacementKind && len(clusterNames) == 1 {
```

Comment:
I don't think it is safe to assume that the primary cluster will be the host cluster.  A target deployment may well not join the host cluster.  I'm not saying you have to fix it in this PR, but I think a TODO comment is warranted so that determination of whether a member cluster is the host cluster is eventually implemented.